src/components/CollapseBar.js

```diff
@@ -12,13 +12,13 @@ import FishIcon from "../images/fish.png";
 
 const ResponsiveAppBar = () => {
     return (
-        <AppBar style={{background: "#1dd1a1", color:"#222f3e"}} position="static">
+        <AppBar style={{background: "#023E8A", color:"#222f3e"}} position="static"> {/* #1dd1a1 */}
```

Comment:
I imagine this file is created from a template and not manually? If manually, what's the `{/* #1dd1a1 */}` ?

Reply:
My bad - it's the color code for the old collapse bar. I commented it so that I wouldn't lose it while testing other blues. I don't think we'll need it anymore, so I deleted.